openrc

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
This file contains all parameter which can be set for deployment scripts.

It can be useful if we are going to deploy env manually. In these case we can set required parameters in this file and use 'source openrc' command to apply it. As result there is no need to search all possible variables in scripts.